subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -168,9 +168,37 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
 
                 reservation = TestsBasedDevicesReservationConfiguration().apply {
                     device = Emulator27
-                    maximum = 50
-                    minimum = 2
-                    testsPerEmulator = 3
+                    maximum = 1
+                    minimum = 1
+                    testsPerEmulator = 1
+                }
+            }
+        }
+    }
+
+    configurationsContainer.register("uiDebug") {
+        tryToReRunOnTargetBranch = false
+        reportSkippedTests = false
+        rerunFailedTests = false
+        reportFlakyTests = false
+        prefixFilter = "com.avito.android.ui.test.AppBarTest"
```

Comment:
Can we not use filtering?
It's easy to forget about this filter

Reply:
Yes. Of course. I'll remove it. But running all tests is not a good idea. We need to design the mechanism if running specified tests. And this mechanism should be convenient for our users.

Reply:
Fixed